roles/pulp_common/handlers/restart_all_pulp.yml

```
@@ -17,12 +17,22 @@
 - name: Get services
   service_facts:
 
+- name: Get services to exclude
+  set_fact:
+    services_to_exclude:
+      - 'pulpcore-worker@.service'
+      - "{{ __pulp_version is version('3.16', '>=') | ternary('pulpcore-resource-manager.service', '') }}"
+
+- name: Listing Pulp services to exclude
+  debug:
+    var: services_to_exclude
```

```suggestion
    verbosity: 1
```
This will make it only output when there is at least debug level 1.

We may need to increase the debug level for CI/molecule. I've been meaning to look into it.

I am OK with us leaving this as is if we can not figure it out.